tox.ini

```
@@ -1,6 +1,7 @@
 [tox]
 envlist =
-    py{38,39,310}-django{32,40,41}
+    py{38,39,310}-django{32,41,42}
+    py{38,39,310,311}-django{41,42}
```

```suggestion
    py{38,39,310}-django{32,41,42}
    py311-django{41,42}
```
jouw versie resulteert in dubbele builds van py38-django41, bijvoorbeeld.